docker-compose-developer.yml

```diff
@@ -3,6 +3,9 @@ version: "3.9"
 x-labels: &oncall-labels
   - "com.grafana.oncall.env=dev"
 
+x-user: &oncall-user
+  "appuser"
```

Comment:
is this needed? according to the `docker-compose` configuration [docs](https://docs.docker.com/compose/compose-file/05-services/#user):

> `user` overrides the user used to run the container process. Default is that set by image (i.e. Dockerfile `USER`), if not set, `root`. 

You are setting `USER` in `engine/Dockerfile` [here](https://github.com/grafana/oncall/pull/2053/files#diff-a25c2322baf692c37d4ecc9c3611772e78dfb8bbaf9cc4ef27b507e60cb3ddc9R57), so I think this may be redundant

Reply:
ahh yes, that makes sense. thanks for the detailed explanation here 👍